Scripts/DCS-BIOS/lib/modules/aircraft_modules/M-2000C.lua

```diff
@@ -0,0 +1,905 @@
+module("M-2000C", package.seeall)
+
+local Module = require("Module")
+
+--- @class M_2000C: Module
+local M_2000C = Module:new("M-2000C", 0x7200, { "M-2000C" })
+
+--v1.39 by Ergo,Matchstick,MisterKnife,WarLord,Espresso29470
+
+--remove Arg# Pilot 1000
+--Get Displays Functions
+
+-- parse radios
+local vhfFrequency = ""
+local uhfFrequency = ""
+
+M_2000C:addExportHook(function()
+	local radios = M_2000C.parse_indication(7)
+	if not radios then
+		return
+	end
+
+	local vhf = coerce_nil_to_string(radios.text_COM_UHF1)
+	if vhf then
+		vhfFrequency = vhf:sub(1, 3) .. vhf:sub(5, 6)
+	end
+
+	local uhf = coerce_nil_to_string(radios.text_COM_UHF2)
+	if uhf then
+		uhfFrequency = uhf:sub(1, 3) .. uhf:sub(5, 6)
+	end
+end)
+
+-- parse fuel
+local fuelFlow = ""
+local fuelJauge = ""
+local fuelTotal = ""
+local FUEL_FLOW_LEN = 3
+
+M_2000C:addExportHook(function(dev0)
+	local fuel = M_2000C.parse_indication(3)
+	if not fuel then
+		return
+	end
```

Comment:
This isn't needed - parse_indication no longer returns `nil`
```suggestion
```

Reply:
```
--- Parses a dcs indication from a string into a key-value table, or nil if no data is available
--- Values are separated with "-----------------------------------------\n"
```
I guess both of those are wrong?

Reply:
2nd line should maybe read parses string with values separated by "---------\n"

Reply:
Yeah the documentation is incorrect there - it won't return nil. But the signature is correct - `{[string|integer]: string}`. I agree the second line could be clearer to mention that is how it parses the string.